### ISSUE TITLE
[Feature Request] Breadcrumb navigation

### ISSUE THREAD

**Issue by larsb24** [anyproto/anytype-ts#831]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing requests and didn't find any that were similar
- [X] I have considered creating a pull request instead and want to proceed

### Clear and concise description of the problem

_Feature doesn't exist yet_

### Suggested solution

Breadcumb navigation on the top of a page to easily navigate upwards when using a lot of nested pages.

<img width="400" alt="OMR_Breadcrumbs_grafische-Darstellung" src="https://github.com/user-attachments/assets/541bf3fe-2dfd-435c-a6df-45aca40778bf">
(Examle of different breadcrumb styles)

I believe it would really improve the navigation experience as well as help the visual understanding of where in a tree of documents one is located.

**Comment by chungusg** [comment]
I feel like this is a simple thing to add that would make anytype absolutely perfect to me. 
Even though you can navigate through the sidebar widget, having breadcrumbs is much more straightforward and cuts out the clutter of seeing every other page in the hierarchy.
Also, I think this is necessary because (unless I'm mistaken) the current page you are on isn't indicated anywhere (not even on the sidebar widget by text color or highlighting), so if you have multiple pages with the same name....how do you even figure out where it's located?

**Comment by chungusg** [comment]
I feel like this is something that I would find really useful, but would be fundamentally hard to implement because of the structure of anytype. I had to realize that because folders don't exist, there isn't really a strict hierarchical structure, which I believe is to maximize the flexibility of use. But this means that there isn't a set location for any object/page, only backlinks and such, so there isn't a breadcrumb trail that would work. It'd be like trying to map out Wikipedia.

I think having at least a "History" type breadcrumb trail would be really nice, though; so you can at least retrace your steps.

**Comment by larsb24** [comment]
That is true. Documents can be linked to anything from anywhere. I think the history type would be nice if you have accessed a page through links on other pages because if you are like 4 pages down it'd be much easier to go back via a breadcrumb than having to press the back button however many times.

**Comment by alensiljak** [comment]
The back button seems to work in the current version (0.40.19 on Android).
And, since breadcrumbs don't really make sense in AnyType, this request is fulfilled?

**Comment by michaeljk2** [comment]
It works in Notion and notion is basically the same

**Comment by Arche151** [comment]
This would be a great addition and something that keeps me from switching from Notion.

And forgive me, if I'm being ignorant, but wouldn't it be quite easy to implement, because you basically just need to render at the top of the page, what you can already see in left panel? 

<img width="267" height="102" alt="Image" src="https://github.com/user-attachments/assets/0d50bbcb-3250-414c-b189-62416257a9c5" />
